routes/UserRoutes.js

```diff
@@ -0,0 +1,45 @@
+const Routes = require('./Routes');
+const UserLogic = require('../logic/UserLogic');
+
+class UserRoutes extends Routes {
+    constructor(app) {
+        super();
+        this.init(app);
+    }
+
+    init(app) {
+        app.post('/createUser', this.RequestBodySchemaValidator.validate(true), async (req, res) => {
```

Comment:
If you are following REST methodology you should rename it to `app.post('/users'`

Reply:
Actually, /user